cdk/lib/frontend.ts

```diff
@@ -124,15 +135,8 @@ export class Frontend extends GuStack {
       },
       monitoringConfiguration: {
         snsTopicName: `alarms-handler-topic-${this.stage}`,
-        http5xxAlarm: {
-          alarmName: alarmName("support-frontend is returning 5XX errors"),
-          alarmDescription: alarmDescription(
-            "Some or all actions on support website are failing"
-          ),
-          actionsEnabled: shouldEnableAlarms,
-          tolerated5xxPercentage: 5,
-        },
-        unhealthyInstancesAlarm: true,
+        http5xxAlarm: http5xxAlarm,
```

Comment:
What happens here when the `http5xxAlarm` has value false?

Reply:
It just doesn't create one.